### PR TITLE
pkgs/development: stdenv.lib -> lib

### PR DESCRIPTION
**Pull request by prusnak**
Part of https://github.com/NixOS/nixpkgs/issues/108938

I inspected files in pkgs/development and took care of folders with 10 or less replacements needed. Others will follow in separate pull requests.

this takes care of the following folders in pkgs/development:
* arduino
* chez-modules
* go-packages
* guile-modules
* idris-modules
* perl-modules
* r-modules
* ruby-modules

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by siraben — APPROVED** [review]
LGTM

**Comment by sternenseemann** [comment]
Eval failure in androidndk-pkgs: <https://gist.github.com/GrahamcOfBorg/27a98d4fdb81bfb21ce9936987df603d>

**Comment by prusnak** [comment]
> Eval failure in androidndk-pkgs: https://gist.github.com/GrahamcOfBorg/27a98d4fdb81bfb21ce9936987df603d

Hopefully addressed correctly in 33ab5be25589f1a276078ab203d76b7ee3bc2b08

**Comment by prusnak** [comment]
Hm, not sure how to fix `androidndk-pkgs`, dropping from the PR

File: pkgs/development/java-modules/maven-minimal.nix [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{ stdenv, pkgs }:
+{ stdenv, lib, pkgs }:
```
Comment (SuperSandro2000) [inline pkgs/development/java-modules/maven-minimal.nix:1]: We should really remove pkgs here.

**Comment by prusnak** [comment]
Removing also java-modules from the PR (my Java kung-fu is not that strong).

File: pkgs/development/java-modules/maven-minimal.nix (hunk 1 above)
Comment (prusnak) [reply pkgs/development/java-modules/maven-minimal.nix]: Removed Java stuff from the PR completely.

Comment (sternenseemann) [reply pkgs/development/java-modules/maven-minimal.nix]: > We should really remove pkgs here.

Side note: I don't think it's wise introducing unrelated changes to these kinds of PRs.

Comment (SuperSandro2000) [reply pkgs/development/java-modules/maven-minimal.nix]: I did not mean in this PR. General in another one.

**Pull request merged**